lib/browser/client-scripts/calibrate.js

```diff
@@ -48,10 +48,13 @@
     }
 
     function getBrowserFeatures() {
+        var iOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
+
         var features = {
             needsCompatLib: needsCompatLib(),
             pixelRatio: window.devicePixelRatio,
-            innerWidth: window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth
+            // use screen.width for iOS, because innerWidth is always 980
```

Comment:
can you provide more information about it (with examples)?
I tried to reproduce your behavior with iPhone simulator and get

```js
{
    ua: 'Mozilla/5.0 (iPhone; CPU iPhone OS 11_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.0 Mobile/15E148 Safari/604.1',
    screenWidth: 375, // window.screen.width
    innerWidth: 375 // window.innerWidth
}
```
`screen.width` and `innerWidth` are the same for me

Reply:
Sure. I guess it depends on the timing. I have put a console.log into the calibrate.js file and got the `980`. When the page is loaded and I enter `innerWidth` in the console, I get the correct value, e.g. `375`.

It's also described here: https://stackoverflow.com/questions/4629969/ios-return-bad-value-for-window-innerheight-width.

Reply:
> It's also described here: https://stackoverflow.com/questions/4629969/ios-return-bad-value-for-window-innerheight-width.

but it was discussed 8 years ago :)
can you provide your BrowserStack configuration?

Reply:
ping

Reply:
Sorry for the delay.

If I set this meta tag:
```
<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">

or as Gemini is supposed to do it:

<meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no">
```
`innerWidth` is correct. See screenshots here: https://www.browserstack.com/screenshots/e3f182f8383f50db559aae3af1e8179ce81e3aa0

If I don't set that meta tag `innerWidth` is incorrect in most cases (980). See screenshots here: https://www.browserstack.com/screenshots/7f6036afa294f3eea57a26e682d7fb0f4c665eb7

I'm not sure, why I got the incorrect results despite Gemini setting the meta tag. I might check that again the next time I have a project involving Gemini. Please close the issue.

Reply:
ok, thx